functions/__fish_apple_touchbar_bind_key.fish

```diff
@@ -1,8 +1,12 @@
 function __fish_apple_touchbar_bind_key --argument-names fn_number fn_text fn_command bind_option
     __fish_apple_touchbar_print_key "SetKeyLabel=F$fn_number=$fn_text"
 
+    if bind -M insert >/dev/null 2>/dev/null
```

Comment:
```suggestion
    if bind -M insert >/dev/null 2>&1
```

Reply:
I proposed a different command to avoid repetition of `/dev/null`.
See https://unix.stackexchange.com/a/80632/362684 for reference.